lib/controllers/ServiceFabrikAdminController.js

```diff
@@ -623,6 +601,75 @@ class ServiceFabrikAdminController extends BaseController {
         .send({}));
   }
 
+  startMaintenance(req, res) {
+    logger.info('req.body=>', req.body);
+    return maintenanceManager
+      .getMaintenaceInfo()
+      .then(body => {
+        if (body === null) {
+          return maintenanceManager
+            .startMaintenace(req.body, req.user)
+            .then(body => res
+              .status(201)
+              .send(body));
+        } else {
+          res
+            .status(403)
+            .send({
+              errorMessage: 'System already in maintenance mode',
+              maintenanceInfo: body
+            });
+        }
+      });
+  }
+
+  updateMaintenance(req, res) {
+    return maintenanceManager
+      .getMaintenaceInfo()
+      .then(body => {
+        if (body !== null) {
+          return maintenanceManager
+            .updateMaintenace(req.body.progress, req.body.state, req.user)
+            .then(body => res
+              .status(200)
+              .send(body));
+        } else {
+          res
+            .status(403)
+            .send({
+              errorMessage: 'System not in maintenance mode'
```

Comment:
Better to define error messages in some constants.

Reply:
As discussed HTTP Status codes are used in similar fashion in all controllers and just changing in couple of methods would lead to confusion to external reviewers. Leaving this as-is as agreed.